tests/test_tpt_reports.py

```diff
@@ -135,3 +135,29 @@ def test_domain_validation():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
             assert return_code == 2, "main() should return with error return code 2"
+
+
+def test_parse_json():
+    """Validate parse json functionality."""
+    payloads_list = []
+    payloads_meta = {}
+    payload_list_test = [
+        {
+            "Payload": "Test payload",
+            "C2 Protocol": "c2_1",
+            "Border Protection": "Not Blocked",
+            "Host Protection": "Blocked",
```

Comment:
In cases where key order doesn't matter, we always prefer to alphabetize:
```suggestion
            "Border Protection": "Not Blocked",
            "C2 Protocol": "c2_1",
            "Host Protection": "Blocked",
            "Payload": "Test payload",
```